JobToCandidateRecs.md

```diff
@@ -0,0 +1,73 @@
+Job To Candidate Recommendation Service
+=========================================
+
+- [Summary](#summary)
+- [Request Response Structure](#request-response-structure)
+- [Table Structure](#table-structure)
+
+## Summary
+DSAD's job-to-candidate-recs service is a FastAPI based web service, which takes a `job_id` and its related `latitude` 
+and `longitude`, and will return a list of `Candidate` within 50 miles of `job_id` location. 
+
+The related dynamodb tables are `JobCandidateGeo` and `CaroteneCandidateGeo`. 
+
+If the `job_id` exists in table `JobCandidateGeo`, the service will grab all items with this `job_id` and collect their `Candidate` attribute as response. If the `job_id` is not found in table `JobCandidateGeo`, the service will try get job's carotene code by calling jobdetails service, and then do a radius search in table `CaroteneCandidateTable` and get candidates within 50 miles of requested job's location for the corresponding `carotene_code`. The service will also add those candidates for the `job_id` to the `JobCandidateGeo` table for future use.
```

Comment:
I would leave this whole section out as well since it is also an implementation detail.